src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -112,7 +112,7 @@ def set_package_assessment_status(self, package_names, package_versions, classif
                 if patch_id == self.__assessment_packages[i]['patchId']:
                     patch_already_saved = True
                     self.__assessment_packages[i]['classifications'] = [classification]
-                    self.__assessment_packages[i]['patchState'] = status
+                    # self.__assessment_packages[i]['patchState'] = status
```

Comment:
Why?

Reply:
`patchState` is deprecated based on the OS-agnostic design spec. On line 123, you can see that it is also commented out, and this matches it. No code in the repo currently uses `patchState`.